src/components/Skills/Skills.tsx

```
@@ -0,0 +1,48 @@
+"use client";
+import { SkillsData } from "@/db/main";
+import { Variants, motion } from "framer-motion";
+import React from "react";
+import styles from "../../styles/components/Skills.module.scss";
+import Skill from "./Skill";
+type SkillsProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)